File: ModelsApp/Models/Filter.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace ModelsApp.Models
{
	public class Filter
	{
		public string FieldName { get; set; }
		public string Operator { get; set; }
		public string FieldValue { get; set; }
	}
}

File: FrontEndApp/src/app/_models/filter.ts
export class Filter {
    public FieldName: string = "";
	public Operator: string = "";
	public FieldValue: string = "";

	constructor(fieldName: string, operator: string, fieldValue: string)
	{
		this.FieldName = fieldName;
		this.Operator = operator;
		this.FieldValue = fieldValue;
	}
}